libs/design-system/brand-logo/src/lib/brand-logo.component.html

```diff
@@ -0,0 +1,10 @@
+<a
+  href="https://humanatlas.io/"
+  target="_blank"
+  [inlineSVG]="'assets/logo/hra-brand-logo.svg'"
```

Comment:
@axdanbol does this respect `app-href`?

Reply:
No, @bhushankhope please add the ` | assetUrl` pipe to make it respect app-href